apps/meteor/app/livechat/server/methods/takeInquiry.js

```diff
@@ -27,7 +27,7 @@ Meteor.methods({
 		}
 
 		const user = Users.findOneOnlineAgentById(Meteor.userId(), {
-			fields: { _id: 1, username: 1, roles: 1, status: 1, statusLivechat: 1 },
+			project: { _id: 1, username: 1, roles: 1, status: 1, statusLivechat: 1 },
```

Comment:
```suggestion
			fields: { _id: 1, username: 1, roles: 1, status: 1, statusLivechat: 1 },
```

This is not a raw model (line 4) 🤔 ... but if you want to make it use a raw model :trollface: that would be amazing

Reply:
done!